README.md

```diff
@@ -134,6 +134,20 @@ You can Get and install the RISC-V toolchain from https://gnutoolchains.com/risc
 
 Go to litex-boards/litex_boards/targets and execute the target you want to build.
 
+For example, build a picorv32 on the ULX3SL:
+```
+cd ./litex-boards/litex_boards/targets
+./ulx3s.py --device LFE5U-85F --cpu-type picorv32
+```
+
+The soft CPU is called `top.bit` and should be in:
```

Comment:
This isn't quite right, the "gateware which contains the SoC (which itself contains the soft CPU and peripherals)"